src/routes/index.test.js

```diff
@@ -10,5 +10,5 @@ describe('Users route', () => {
         .end((err, res) => {
             expect(res.status).to.eq(200)
         })
-    })
+    }) n
```

Comment:
### Nitpick

I think this was added by mistake 😅

Reply:
lol that would be correct. Thanks 😄